Mathlib/Util/AssertExists.lean

```diff
@@ -22,6 +23,66 @@ Implement `assert_instance` and `assert_no_instance`
 section
 open Lean Elab Meta Command
 
+namespace Mathlib.AssertNotExist
+
+/-- `#check_assertions` retrieves all declarations and all imports that were declared
+not to exist so far (including in transitively imported files) and reports their current
+status:
+* ✓ means the declaration or import exists,
+* × means the declaration or import does not exist.
+
+This means that the expectation is that all checks *succeed* by the time `#check_assertions`
+is used, typically once all of `Mathlib` has been built.
+
+If all declarations and imports are available when `#check_assertions` is used,
+then the command logs an info. Otherwise, it emits a warning.
+
+The variant `#check_assertions!` only prints declarations/imports that are not present in the
+environment.  In particular, it is silent if everything is imported, making it useful for testing.
+-/
+elab "#check_assertions" tk:("!")?: command => do
+  let env ← getEnv
+  let ext := assertExistsExt.getState env
```

Comment:
I like this suggestion and added `getSortedAssertExists` to the extension file.  I also moved there the `addDeclEntry` function: I did not place it there before, since I thought that this was what causing the `lake build` issues.  I had not realized that the issue was the `olean` explosion.